kernel: bump 5.4 to 5.4.123

graysky2 opened the pull request:
All patches automatically rebased.

Build system: x86_64
Build-tested: ipq806x/R7800
Run-tested: ipq806x/R7800

No dmesg regressions, everything functional

Signed-off-by: John Audia <graysky@archlinux.us>

graysky2 committed:
kernel: bump 5.4 to 5.4.122

All patches automatically rebased.

Build system: x86_64
Build-tested: ipq806x/R7800
Run-tested: ipq806x/R7800

No dmesg regressions, everything functional

Signed-off-by: John Audia <graysky@archlinux.us>

zuzia-dev commented:
Build system: x86_64
Build-tested: lantiq/xrx200/Easybox 904 xDSL (VGV952CJW33-E-IR)
Run-tested: lantiq/xrx200/Easybox 904 xDSL (VGV952CJW33-E-IR)

OK.

graysky2 committed:
kernel: bump 5.4 to 5.4.123

All patches automatically rebased.

Build system: x86_64
Build-tested: ipq806x/R7800
Run-tested: ipq806x/R7800

No dmesg regressions, everything functional

Signed-off-by: John Audia <graysky@archlinux.us>

hauke commented:
Thank you for the patches, I applied them to master.
